src/App.sass

```diff
@@ -0,0 +1,53 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600;900&display=swap')

```

Comment:
Use `<link>` into `index.html` for loading web-fonts. It's all because of `@import` defer the loading of the included resource until the file is fetched. For more details you could read [this](https://web.dev/optimize-webfont-loading/) and [this](https://web.dev/font-best-practices/) articles.